learning_resources_search/indexing_api.py

```diff
@@ -235,62 +232,39 @@ def index_items(documents, object_type, index_types, **kwargs):
                     raise ReindexError(msg)
 
 
-def index_courses(ids, index_types):
+def index_learning_resources(ids, resource_type, index_types):
```

Comment:
Now that we use the same serializer for all learning resources, we don't need a separate index_ function for each learning resource type